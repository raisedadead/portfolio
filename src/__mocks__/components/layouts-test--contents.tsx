import Layout from '@/components/layouts';

const Contents: React.FC = () => {
  return (
    <Layout>
      <section>
        <div className='prose prose-stone max-w-none px-20 pt-5 pb-20'>
          <h1 className='py-2 text-center'>Blog</h1>
          <p>
            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed vel
            maximus lorem, in semper lacus. Praesent scelerisque vestibulum dui,
            eu facilisis dui dapibus vel.
          </p>
          <h2>Consectetur Adipiscing Elit</h2>
          <p>
            Integer fringilla ipsum vitae elit interdum, at fermentum odio
            consequat. Fusce eu felis vel metus commodo dapibus ut nec purus.
          </p>
          <h3>Curabitur Venenatis</h3>
          <ul>
            <li>Fusce eget ante vel lacus consequat laoreet.</li>
            <li>Mauris vitae magna sed odio cursus sodales.</li>
            <li>Phasellus semper enim sit amet ultricies consectetur.</li>
          </ul>
          <h2>Donec Velit Neque</h2>
          <p>
            Donec velit neque, auctor et enim in, condimentum fermentum nisi.
            Morbi lacinia velit justo, nec tempus nulla porta quis. Vivamus
            dictum, risus at feugiat pharetra, erat sem cursus felis, id
            consectetur ante ipsum ac mi.
          </p>
          <h3>Quisque Elementum</h3>
          <ol>
            <li>Etiam commodo bibendum justo.</li>
            <li>Pellentesque non semper diam, nec convallis lacus.</li>
            <li>Integer in volutpat nunc, at ullamcorper erat.</li>
          </ol>
          <h2>Nunc Suscipit</h2>
          <table className='border'>
            <thead>
              <tr>
                <th>Header 1</th>
                <th>Header 2</th>
                <th>Header 3</th>
              </tr>
            </thead>
            <tbody>
              <tr>
                <td>Row 1, Cell 1</td>
                <td>Row 1, Cell 2</td>
                <td>Row 1, Cell 3</td>
              </tr>
              <tr>
                <td>Row 2, Cell 1</td>
                <td>Row 2, Cell 2</td>
                <td>Row 2, Cell 3</td>
              </tr>
              <tr>
                <td>Row 3, Cell 1</td>
                <td>Row 3, Cell 2</td>
                <td>Row 3, Cell 3</td>
              </tr>
            </tbody>
          </table>
          <h3>Etiam Malesuada</h3>
          <p>
            Etiam malesuada sapien ut cursus convallis. Sed ullamcorper
            facilisis justo, et posuere elit convallis a. Pellentesque habitant
            morbi tristique senectus et netus et malesuada fames ac turpis
            egestas.
          </p>
          <h2>Integer Vitae</h2>
          <p>
            Integer vitae ipsum vitae lacus lacinia, id congue justo convallis.
            Cras auctor sapien vitae turpis consequat ultrices. Fusce eu odio
            lorem. Duis scelerisque purus ac justo pharetra, eget tincidunt
            neque fringilla. Aenean nec ante turpis. Nulla facilisi.
          </p>
          <h3>Maecenas Tincidunt</h3>
          <ul>
            <li>Nulla non ligula non diam consequat consequat.</li>
            <li>Praesent quis massa in sem consequat aliquam.</li>
            <li>Ut ullamcorper libero at blandit fermentum.</li>
          </ul>
          <h2>Aliquam Eros</h2>
          <p>
            Aliquam eros magna, ullamcorper et imperdiet id, euismod in mi.
            Fusce rhoncus metus ut ipsum posuere, sit amet porttitor est
            dapibus.
          </p>
          <h3>Vestibulum Dapibus</h3>
          <ol>
            <li>Vivamus euismod nunc vel lorem ullamcorper.</li>
            <li>Curabitur vehicula libero vel mi dapibus.</li>
            <li>In hac habitasse platea dictumst.</li>
          </ol>
          <h2>Sed Egestas</h2>
          <table className='border'>
            <thead>
              <tr>
                <th>Column A</th>
                <th>Column B</th>
                <th>Column C</th>
              </tr>
            </thead>
            <tbody>
              <tr>
                <td>Cell A1</td>
                <td>Cell B1</td>
                <td>Cell C1</td>
              </tr>
              <tr>
                <td>Cell A2</td>
                <td>Cell B2</td>
                <td>Cell C2</td>
              </tr>
              <tr>
                <td>Cell A3</td>
                <td>Cell B3</td>
                <td>Cell C3</td>
              </tr>
            </tbody>
          </table>
          <h3>Fusce Varius</h3>
          <p>
            Fusce varius velit ac sapien malesuada, a tincidunt purus consequat.
            Nunc luctus dapibus justo, id consequat quam tincidunt vitae.
            Suspendisse potenti.
          </p>
          <h2>Phasellus Eleifend</h2>
          <p>
            Phasellus eleifend nulla ut sollicitudin blandit. Maecenas et
            bibendum erat, eu sollicitudin nisl. Integer porttitor libero id sem
            consectetur, eu accumsan est cursus.
          </p>
          <h4>End.</h4>
        </div>
      </section>
    </Layout>
  );
};

export default Contents;
