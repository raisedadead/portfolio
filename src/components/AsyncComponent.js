import React, { Component } from 'react'
import { DoubleBounce } from 'better-react-spinkit'
import { StylesHelper } from './Utils'

const content = StylesHelper('uk-position-center', 'uk-text-center')
export default function asyncComponent(importComponent) {
  class AsyncComponent extends Component {
    constructor(props) {
      super(props)
      this.state = { component: null }
    }

    async componentDidMount() {
      const { default: component } = await importComponent()
      this.setState({ component: component })
    }

    render() {
      const C = this.state.component
      return C ? (
        <C {...this.props} />
      ) : (
        <DoubleBounce size={50} className={content} />
      )
    }
  }

  return AsyncComponent
}
