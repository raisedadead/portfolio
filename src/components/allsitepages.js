import React from 'react'
import { Link } from 'gatsby'
const pages = require(`../../nav-config`).pages

class AllSitePages extends React.Component {
	render() {
		let listItems = []
		for (const page in pages) {
			if (pages.hasOwnProperty(page)) {
				if(pages[page].navTitle){
					listItems.push(
						<li key={page} id={page}>
							<Link activeClassName='current' to={pages[page].path}>
								{pages[page].navTitle}
							</Link>
						</li>
					)
				}
			}
		}
			return (
		<section>
			<h2>About this&nbsp;Starter</h2>
			<nav>
				<ul>
					{listItems}
				</ul>
			</nav>
	  </section>
    )
  }
}

export default AllSitePages
