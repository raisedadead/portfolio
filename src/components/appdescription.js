import React from 'react'
import { OutboundLink } from 'gatsby-plugin-google-analytics'

class AppDescription extends React.Component {
	render() {
		return (
			<section>
				<p data-testid="app-description">This is a Gatsbyjs starter that I use to test things and practice.  You are welcome to look around for examples how to use many Gatsby plugins and utilities.</p>
				<code className='block'>
					$ gatsby new my-website https://github.com/DavidSabine/gatsby-starter<br />
					# these commands presume that you have node, yarn, git, and gatsby-cli<br />
					# visit the readme at <OutboundLink href="//github.com/DavidSabine/gatsby-starter/blob/master/README.md">github.com/DavidSabine/gatsby-starter</OutboundLink>
				</code>
				<p>To see it all in action, then run the following:</p>
				<code className='block'>
					$ yarn<br />
					$ yarn develop
					# then go to http://localhost:8000 in your browser<br />
				</code>
				<p>To see the tests run, use...</p>
				<code className='block'>
					$ yarn test<br />
					$ yarn test:e2e<br />
				</code>
				<p>
					The demo of this starter is at <OutboundLink href="//gatsby-starter.davesabine.com">gatsby-starter.davesabine.com</OutboundLink> and the full source code is at <OutboundLink href="//github.com/DavidSabine/gatsby-starter">github.com/DavidSabine/gatsby-starter</OutboundLink>.
				</p>
			</section>
		)
	}
}

export default AppDescription
