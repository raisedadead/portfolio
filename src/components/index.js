import { PageHeaders } from './PageHeaders'
import { MainNav, SocialNav } from './Nav'
import { Footer } from './Footer'
import { Background } from './Background'

import { ImageLoader, StylesHelper } from './Utils/'

export {
  PageHeaders,
  MainNav,
  SocialNav,
  Footer,
  Background,
  ImageLoader,
  StylesHelper
}
