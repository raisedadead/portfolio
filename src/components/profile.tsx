import { CustomLink as Link } from './custom-link';
import { Social } from './social';

export const Profile: React.FC = () => {
  return (
    <div className="flex flex-col items-center justify-center py-12 text-center">
      <div className="mb-8 md:mb-10">
        <img
          id="profile-image"
          alt="Mrugesh Mohapatra's profile picture."
          src="/images/profile.small.webp"
          className="h-28 w-28 border-4 border-orange-50 shadow-[4px_4px_0px_0px_rgba(0,0,0,1)] md:h-36 md:w-36"
          width={144}
          height={144}
        />
      </div>
      <div className="relative mb-8 flex -rotate-12 transform flex-col items-center md:mb-10">
        <h1 className="group rotate-12 transform cursor-pointer text-3xl font-extrabold text-slate-800 md:text-5xl">
          {'Mrugesh Mohapatra'.toLowerCase()}
          <span
            className="absolute left-1/4 hidden w-auto rounded-md bg-black p-3 text-sm font-normal text-white group-hover:block"
            style={{
              top: '100%',
              marginTop: '0.75rem',
              transform: 'translateX(-50%)',
            }}
          >
            <Link
              href="https://itinerarium.github.io/phoneme-synthesis/?w= /ˈm.ruː.geɪ.ʃ/"
              className="no-underline"
              aria-label="Pronunciation of my name"
            >
              🗣 /ˈm.ruː.geɪ.ʃ/
            </Link>
            <svg
              className="absolute left-1/4 h-4 w-full text-black"
              style={{ top: '-0.5rem', transform: 'translateX(-50%)' }}
              viewBox="0 0 255 255"
              xmlSpace="preserve"
              aria-hidden="true"
            >
              <polygon className="fill-current" points="128,0 128,128 0,128" />
            </svg>
          </span>
        </h1>
        <div className="m-1 mx-auto -mt-8 w-2/5 border-8 border-orange-50 bg-orange-50" />
      </div>
      <h2 className="mb-4 max-w-md p-1 text-xl leading-relaxed font-bold text-slate-700 md:text-2xl">
        nocturnal developer 🦉 • open-source enthusiast 🌏 • photography noob 📷
      </h2>
      <h3 className="mb-8 p-1 text-lg leading-relaxed font-medium text-slate-700 md:text-xl">
        Principal Maintainer — Cloud Infrastructure & Open-source,{' '}
        <Link
          href="https://www.freecodecamp.org/news/team#:~:text=around%20the%20world.-,Mrugesh%20Mohapatra,-from%20Bengaluru%2C%20India"
          className="-ml-1 p-1 text-slate-700 underline decoration-orange-200 decoration-wavy decoration-2 underline-offset-2 hover:text-white hover:decoration-white"
          aria-label="freecodecamp.org"
        >
          freeCodeCamp.org
        </Link>
      </h3>

      <div className="mb-10 flex flex-row items-center justify-center space-x-4">
        <Link
          aria-label="Book a 1-on-1 Call"
          className="h-14 w-80 border-2 border-black bg-orange-50 p-3 text-lg font-semibold text-black shadow-[2px_2px_0px_rgba(0,0,0,1)] transition-all duration-200 hover:bg-gray-700 hover:text-white hover:shadow-none active:bg-black active:shadow-none"
          href="https://topmate.io/mrugesh"
          type="button"
        >
          <span className="inline-flex items-center">Get in touch!</span>
        </Link>
        <Link
          aria-label="Browse my blog"
          className="h-14 w-36 border-2 border-black bg-orange-200 p-3 text-lg font-semibold text-black shadow-[2px_2px_0px_rgba(0,0,0,1)] transition-all duration-200 hover:bg-gray-700 hover:text-white hover:shadow-none active:bg-black active:shadow-none"
          href="https://hn.mrugesh.dev"
          type="button"
        >
          <span className="inline-flex items-center">Blog</span>
        </Link>
      </div>
      <div className="prose prose-lg prose-slate mx-auto mt-8 max-w-3xl">
        <h3 className="mb-4 text-center font-bold text-slate-700">Elsewhere on the internet</h3>
        <Social />
      </div>
    </div>
  );
};
