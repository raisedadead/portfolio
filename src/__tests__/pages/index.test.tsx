import { Profile } from '@/components/profile';
import { render, screen } from '@testing-library/react';
import { describe, expect, it, vi } from 'vitest';

vi.mock('@/components/profile', () => ({
  Profile: vi.fn(() => <div data-testid='profile-component'>Profile Component</div>)
}));

vi.mock('@/lib/utils', () => ({
  cn: (...classes: string[]) => classes.filter(Boolean).join(' ')
}));

describe('Index Page', () => {
  it('renders main section with profile component', () => {
    const { container } = render(
      <section className='mb-12'>
        <div className='mx-auto max-w-4xl'>
          <Profile />
        </div>
      </section>
    );

    expect(container.querySelector('section')).toHaveClass('mb-12');
    expect(container.querySelector('div')).toHaveClass('mx-auto', 'max-w-4xl');
    expect(screen.getByTestId('profile-component')).toBeInTheDocument();
  });

  it('profile component is rendered within container', () => {
    const { container } = render(
      <section className='mb-12'>
        <div className='mx-auto max-w-4xl'>
          <Profile />
        </div>
      </section>
    );

    const profileContainer = container.querySelector('.max-w-4xl');
    expect(profileContainer).toContainElement(screen.getByTestId('profile-component'));
    expect(vi.mocked(Profile)).toHaveBeenCalled();
  });
});
