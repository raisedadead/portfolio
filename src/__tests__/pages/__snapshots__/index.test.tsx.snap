// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Home > renders a snapshot of the homepage 1`] = `
<DocumentFragment>
  <div
    class="container"
  >
    <div
      class="fixed inset-0 -z-10"
    >
      <div
        class="relative h-screen w-screen overflow-hidden bg-gradient-to-b from-emerald-300 via-orange-200 to-teal-200"
      >
        <svg
          class="absolute bottom-0 left-0 w-full"
          preserveAspectRatio="none"
          viewBox="0 0 1000 800"
        >
          <path
            d="M0 800 C 200 750, 300 850, 500 800 S 700 750, 1000 800 V 800 H 0 Z"
            fill="rgba(50, 222, 212, 0.1)"
          />
          <path
            d="M0 795 C 200 750, 300 840, 500 795 S 700 750, 1000 795 V 800 H 0 Z"
            fill="rgba(50, 222, 212, 0.15000000000000002)"
          />
          <path
            d="M0 790 C 200 750, 300 830, 500 790 S 700 750, 1000 790 V 800 H 0 Z"
            fill="rgba(50, 222, 212, 0.2)"
          />
          <path
            d="M0 785 C 200 750, 300 820, 500 785 S 700 750, 1000 785 V 800 H 0 Z"
            fill="rgba(50, 222, 212, 0.25)"
          />
          <path
            d="M0 780 C 200 750, 300 810, 500 780 S 700 750, 1000 780 V 800 H 0 Z"
            fill="rgba(50, 222, 212, 0.30000000000000004)"
          />
          <path
            d="M0 775 C 200 750, 300 800, 500 775 S 700 750, 1000 775 V 800 H 0 Z"
            fill="rgba(50, 222, 212, 0.35)"
          />
          <path
            d="M0 770 C 200 750, 300 790, 500 770 S 700 750, 1000 770 V 800 H 0 Z"
            fill="rgba(50, 222, 212, 0.4)"
          />
          <path
            d="M0 765 C 200 750, 300 780, 500 765 S 700 750, 1000 765 V 800 H 0 Z"
            fill="rgba(50, 222, 212, 0.45000000000000007)"
          />
        </svg>
        <div
          class="absolute bottom-0 left-0 h-32 w-full bg-gradient-to-t from-teal-600 to-transparent opacity-30"
        />
      </div>
    </div>
    <main
      class="mx-auto my-2 w-[90%] py-8 lg:w-[75%] xl:w-[80%]"
    >
      <nav
        class="m-2 max-h-none pt-2"
      >
        <div
          class="absolute right-4 top-4 text-left"
          data-headlessui-state=""
        >
          <div>
            <button
              aria-expanded="false"
              aria-haspopup="menu"
              class="flex h-10 items-center border-2 border-black bg-orange-200 p-1.5 text-black shadow-[2px_2px_0px_rgba(0,0,0,1)] hover:bg-gray-700 hover:text-white hover:shadow-none focus:outline-none active:bg-black active:shadow-none"
              data-headlessui-state=""
              id="headlessui-menu-button-:rb:"
              type="button"
            >
              <span
                class="sr-only"
              >
                Open navigation menu
              </span>
              <svg
                aria-hidden="true"
                class="h-6 w-6"
                data-slot="icon"
                fill="none"
                stroke="currentColor"
                stroke-width="1.5"
                viewBox="0 0 24 24"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M3.75 5.25h16.5m-16.5 4.5h16.5m-16.5 4.5h16.5m-16.5 4.5h16.5"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                />
              </svg>
            </button>
          </div>
        </div>
        <div
          class="absolute left-4 top-4 text-left"
        >
          <a
            aria-label="Go Home"
            class="flex h-10 items-center border-2 border-black bg-orange-200 p-1.5 text-black shadow-[2px_2px_0px_rgba(0,0,0,1)] hover:bg-gray-700 hover:text-white hover:shadow-none focus:outline-none active:bg-black active:shadow-none"
            href="/"
            role="link"
          >
            <span
              class="sr-only"
            >
              Go Home
            </span>
            <svg
              aria-hidden="true"
              class="h-6 w-6"
              data-slot="icon"
              fill="none"
              stroke="currentColor"
              stroke-width="1.5"
              viewBox="0 0 24 24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="m2.25 12 8.954-8.955c.44-.439 1.152-.439 1.591 0L21.75 12M4.5 9.75v10.125c0 .621.504 1.125 1.125 1.125H9.75v-4.875c0-.621.504-1.125 1.125-1.125h2.25c.621 0 1.125.504 1.125 1.125V21h4.125c.621 0 1.125-.504 1.125-1.125V9.75M8.25 21h8.25"
                stroke-linecap="round"
                stroke-linejoin="round"
              />
            </svg>
          </a>
        </div>
      </nav>
      <div
        class="p-5"
      >
        <div
          class="mx-auto"
        >
          <section
            class="mb-12"
          >
            <div
              class="mx-auto max-w-4xl"
            >
              <div
                class="flex flex-col items-center justify-center py-12 text-center"
              >
                <div
                  class="mb-8 md:mb-10"
                >
                  <img
                    alt="Mrugesh Mohapatra's profile picture."
                    class="h-28 w-28 border-4 border-orange-50 shadow-[4px_4px_0px_0px_rgba(0,0,0,1)] md:h-36 md:w-36"
                    data-nimg="1"
                    decoding="async"
                    fetchpriority="high"
                    height="144"
                    id="profile-image"
                    src="/_next/image?url=%2Fimages%2Fprofile.small.webp&w=384&q=75"
                    srcset="/_next/image?url=%2Fimages%2Fprofile.small.webp&w=256&q=75 1x, /_next/image?url=%2Fimages%2Fprofile.small.webp&w=384&q=75 2x"
                    style="color: transparent;"
                    width="144"
                  />
                </div>
                <div
                  class="relative mb-8 flex -rotate-12 transform flex-col items-center md:mb-10"
                >
                  <h1
                    class="group rotate-12 transform cursor-pointer text-3xl font-extrabold text-slate-800 md:text-5xl"
                  >
                    mrugesh mohapatra
                    <span
                      class="absolute left-1/4 hidden w-auto rounded-md bg-black p-3 text-sm font-normal text-white group-hover:block"
                      style="top: 100%; margin-top: 0.75rem; transform: translateX(-50%);"
                    >
                      <a
                        aria-label=""
                        class="no-underline"
                        href="https://itinerarium.github.io/phoneme-synthesis/?w= /ˈm.ruː.geɪ.ʃ/"
                        rel=" noopener noreferrer"
                        role="link"
                        target="_blank"
                        type=""
                      >
                        🗣 /ˈm.ruː.geɪ.ʃ/
                      </a>
                      <svg
                        class="absolute left-1/4 h-4 w-full text-black"
                        style="top: -0.5rem; transform: translateX(-50%);"
                        viewBox="0 0 255 255"
                        xml:space="preserve"
                      >
                        <polygon
                          class="fill-current"
                          points="128,0 128,128 0,128"
                        />
                      </svg>
                    </span>
                  </h1>
                  <div
                    class="m-1 mx-auto -mt-8 w-2/5 border-8 border-orange-50 bg-orange-50"
                  />
                </div>
                <h2
                  class="mb-4 max-w-md p-1 text-xl font-bold leading-relaxed text-slate-700 md:text-2xl"
                >
                  nocturnal developer 🦉 • open-source enthusiast 🌏 • photography noob 📷
                </h2>
                <h3
                  class="mb-8 p-1 text-lg font-medium leading-relaxed text-slate-700 md:text-xl"
                >
                  Principal Maintainer — Cloud Infrastructure & Open-source, 
                  <a
                    aria-label=""
                    class="-ml-1 p-1 text-slate-700 underline decoration-orange-200 decoration-wavy decoration-2 underline-offset-2 hover:text-white hover:decoration-white"
                    href="https://www.freecodecamp.org/news/team#:~:text=around%20the%20world.-,Mrugesh%20Mohapatra,-from%20Bengaluru%2C%20India"
                    rel=" noopener noreferrer"
                    role="link"
                    target="_blank"
                    type=""
                  >
                    freeCodeCamp.org
                  </a>
                </h3>
                <div
                  class="mb-10 flex flex-col items-center justify-center space-y-4"
                >
                  <a
                    aria-label=""
                    class="h-14 w-80 border-2 border-black bg-orange-50 p-3 text-lg font-semibold text-black shadow-[2px_2px_0px_rgba(0,0,0,1)] transition-all duration-200 hover:bg-gray-700 hover:text-white hover:shadow-none active:bg-black active:shadow-none"
                    href="https://topmate.io/mrugesh"
                    rel=" noopener noreferrer"
                    role="link"
                    target="_blank"
                    type="button"
                  >
                    <span
                      class="inline-flex items-center"
                    >
                      Get in touch!
                    </span>
                  </a>
                  <div
                    class="flex flex-row justify-between space-x-8"
                  >
                    <a
                      aria-label=""
                      class="flex h-12 w-36 items-center justify-center border-2 border-black bg-orange-200 text-base font-medium text-black shadow-[2px_2px_0px_rgba(0,0,0,1)] transition-all duration-200 hover:bg-gray-700 hover:text-white hover:shadow-none active:bg-black active:shadow-none"
                      href="/blog"
                      role="link"
                    >
                      <span>
                        Blog
                      </span>
                    </a>
                    <a
                      aria-label=""
                      class="flex h-12 w-36 items-center justify-center border-2 border-black bg-orange-200 text-base font-medium text-black shadow-[2px_2px_0px_rgba(0,0,0,1)] transition-all duration-200 hover:bg-gray-700 hover:text-white hover:shadow-none active:bg-black active:shadow-none"
                      href="https://bio.link/mrugesh"
                      rel=" noopener noreferrer"
                      role="link"
                      target="_blank"
                      type="button"
                    >
                      <span>
                        Links
                      </span>
                    </a>
                  </div>
                </div>
                <div
                  class="prose prose-lg prose-slate mx-auto mt-8 max-w-3xl"
                >
                  <h3
                    class="mb-4 text-center font-bold text-slate-700"
                  >
                    Elsewhere on the internet
                  </h3>
                  <div
                    class="mx-auto mb-1 mt-2 flex flex-row items-center justify-center space-x-2 space-y-0"
                  >
                    <a
                      aria-label=""
                      class="h-8 w-8 border-2 border-black bg-orange-200 p-1.5 text-black shadow-[2px_2px_0px_rgba(0,0,0,1)] hover:bg-gray-700 hover:text-white hover:shadow-none active:bg-black active:shadow-none"
                      href="https://twitter.com/raisedadead"
                      rel="me noopener noreferrer"
                      role="link"
                      target="_blank"
                      type="button"
                    >
                      <span
                        class="sr-only"
                      >
                        Twitter
                      </span>
                      <svg
                        aria-hidden="true"
                        fill="currentColor"
                        viewBox="0 0 24 24"
                      >
                        <path
                          d="M8.29 20.251c7.547 0 11.675-6.253 11.675-11.675 0-.178 0-.355-.012-.53A8.348 8.348 0 0022 5.92a8.19 8.19 0 01-2.357.646 4.118 4.118 0 001.804-2.27 8.224 8.224 0 01-2.605.996 4.107 4.107 0 00-6.993 3.743 11.65 11.65 0 01-8.457-4.287 4.106 4.106 0 001.27 5.477A4.072 4.072 0 012.8 9.713v.052a4.105 4.105 0 003.292 4.022 4.095 4.095 0 01-1.853.07 4.108 4.108 0 003.834 2.85A8.233 8.233 0 012 18.407a11.616 11.616 0 006.29 1.84"
                        />
                      </svg>
                    </a>
                    <a
                      aria-label=""
                      class="h-8 w-8 border-2 border-black bg-orange-200 p-1.5 text-black shadow-[2px_2px_0px_rgba(0,0,0,1)] hover:bg-gray-700 hover:text-white hover:shadow-none active:bg-black active:shadow-none"
                      href="https://github.com/raisedadead"
                      rel="me noopener noreferrer"
                      role="link"
                      target="_blank"
                      type="button"
                    >
                      <span
                        class="sr-only"
                      >
                        GitHub
                      </span>
                      <svg
                        aria-hidden="true"
                        fill="currentColor"
                        viewBox="0 0 24 24"
                      >
                        <path
                          clip-rule="evenodd"
                          d="M12 2C6.477 2 2 6.484 2 12.017c0 4.425 2.865 8.18 6.839 9.504.5.092.682-.217.682-.483 0-.237-.008-.868-.013-1.703-2.782.605-3.369-1.343-3.369-1.343-.454-1.158-1.11-1.466-1.11-1.466-.908-.62.069-.608.069-.608 1.003.07 1.531 1.032 1.531 1.032.892 1.53 2.341 1.088 2.91.832.092-.647.35-1.088.636-1.338-2.22-.253-4.555-1.113-4.555-4.951 0-1.093.39-1.988 1.029-2.688-.103-.253-.446-1.272.098-2.65 0 0 .84-.27 2.75 1.026A9.564 9.564 0 0112 6.844c.85.004 1.705.115 2.504.337 1.909-1.296 2.747-1.027 2.747-1.027.546 1.379.202 2.398.1 2.651.64.7 1.028 1.595 1.028 2.688 0 3.848-2.339 4.695-4.566 4.943.359.309.678.92.678 1.855 0 1.338-.012 2.419-.012 2.747 0 .268.18.58.688.482A10.019 10.019 0 0022 12.017C22 6.484 17.522 2 12 2z"
                          fill-rule="evenodd"
                        />
                      </svg>
                    </a>
                    <a
                      aria-label=""
                      class="h-8 w-8 border-2 border-black bg-orange-200 p-1.5 text-black shadow-[2px_2px_0px_rgba(0,0,0,1)] hover:bg-gray-700 hover:text-white hover:shadow-none active:bg-black active:shadow-none"
                      href="https://instagram.com/raisedadead"
                      rel="me noopener noreferrer"
                      role="link"
                      target="_blank"
                      type="button"
                    >
                      <span
                        class="sr-only"
                      >
                        Instagram
                      </span>
                      <svg
                        aria-hidden="true"
                        fill="currentColor"
                        viewBox="0 0 24 24"
                      >
                        <path
                          clip-rule="evenodd"
                          d="M12.315 2c2.43 0 2.784.013 3.808.06 1.064.049 1.791.218 2.427.465a4.902 4.902 0 011.772 1.153 4.902 4.902 0 011.153 1.772c.247.636.416 1.363.465 2.427.048 1.067.06 1.407.06 4.123v.08c0 2.643-.012 2.987-.06 4.043-.049 1.064-.218 1.791-.465 2.427a4.902 4.902 0 01-1.153 1.772 4.902 4.902 0 01-1.772 1.153c-.636.247-1.363.416-2.427.465-1.067.048-1.407.06-4.123.06h-.08c-2.643 0-2.987-.012-4.043-.06-1.064-.049-1.791-.218-2.427-.465a4.902 4.902 0 01-1.772-1.153 4.902 4.902 0 01-1.153-1.772c-.247-.636-.416-1.363-.465-2.427-.047-1.024-.06-1.379-.06-3.808v-.63c0-2.43.013-2.784.06-3.808.049-1.064.218-1.791.465-2.427a4.902 4.902 0 011.153-1.772A4.902 4.902 0 015.45 2.525c.636-.247 1.363-.416 2.427-.465C8.901 2.013 9.256 2 11.685 2h.63zm-.081 1.802h-.468c-2.456 0-2.784.011-3.807.058-.975.045-1.504.207-1.857.344-.467.182-.8.398-1.15.748-.35.35-.566.683-.748 1.15-.137.353-.3.882-.344 1.857-.047 1.023-.058 1.351-.058 3.807v.468c0 2.456.011 2.784.058 3.807.045.975.207 1.504.344 1.857.182.466.399.8.748 1.15.35.35.683.566 1.15.748.353.137.882.3 1.857.344 1.054.048 1.37.058 4.041.058h.08c2.597 0 2.917-.01 3.96-.058.976-.045 1.505-.207 1.858-.344.466-.182.8-.398 1.15-.748.35-.35.566-.683.748-1.15.137-.353.3-.882.344-1.857.048-1.055.058-1.37.058-4.041v-.08c0-2.597-.01-2.917-.058-3.96-.045-.976-.207-1.505-.344-1.858a3.097 3.097 0 00-.748-1.15 3.098 3.098 0 00-1.15-.748c-.353-.137-.882-.3-1.857-.344-1.023-.047-1.351-.058-3.807-.058zM12 6.865a5.135 5.135 0 110 10.27 5.135 5.135 0 010-10.27zm0 1.802a3.333 3.333 0 100 6.666 3.333 3.333 0 000-6.666zm5.338-3.205a1.2 1.2 0 110 2.4 1.2 1.2 0 010-2.4z"
                          fill-rule="evenodd"
                        />
                      </svg>
                    </a>
                    <a
                      aria-label=""
                      class="h-8 w-8 border-2 border-black bg-orange-200 p-1.5 text-black shadow-[2px_2px_0px_rgba(0,0,0,1)] hover:bg-gray-700 hover:text-white hover:shadow-none active:bg-black active:shadow-none"
                      href="https://linkedin.com/in/mrugeshm"
                      rel="me noopener noreferrer"
                      role="link"
                      target="_blank"
                      type="button"
                    >
                      <span
                        class="sr-only"
                      >
                        LinkedIn
                      </span>
                      <svg
                        aria-hidden="true"
                        fill="currentColor"
                        viewBox="0 0 24 24"
                      >
                        <path
                          clip-rule="evenodd"
                          d="M19 0h-14c-2.761 0-5 2.239-5 5v14c0 2.761 2.239 5 5 5h14c2.762 0 5-2.239 5-5v-14c0-2.761-2.238-5-5-5zm-11 19h-3v-11h3v11zm-1.5-12.268c-.966 0-1.75-.79-1.75-1.764s.784-1.764 1.75-1.764 1.75.79 1.75 1.764-.783 1.764-1.75 1.764zm13.5 12.268h-3v-5.604c0-3.368-4-3.113-4 0v5.604h-3v-11h3v1.765c1.396-2.586 7-2.777 7 2.476v6.759z"
                          fill-rule="evenodd"
                        />
                      </svg>
                    </a>
                  </div>
                </div>
              </div>
            </div>
          </section>
        </div>
      </div>
      <footer
        class="p-2"
      >
        <div
          class="font-mono text-gray-700 text-sm text-center mx-8 md:mx-auto"
        >
          <p>
            © 2012-2024 Mrugesh Mohapatra Co. — All rights reserved.
          </p>
          <p
            class="mt-2"
          >
            <a
              aria-label=""
              class="text-gray-700 hover:text-black rounded-full hover:bg-white py-1 px-2"
              href="/terms"
              role="link"
            >
              Terms
            </a>
            •
            <a
              aria-label=""
              class="text-gray-700 hover:text-black rounded-full hover:bg-white py-1 px-2"
              href="/privacy"
              role="link"
            >
              Privacy
            </a>
            •
            <a
              aria-label=""
              class="text-gray-700 hover:text-black rounded-full hover:bg-white py-1 px-2"
              href="/refunds"
              role="link"
            >
              Refunds
            </a>
            •
            <a
              aria-label=""
              class="text-gray-700 hover:text-black rounded-full hover:bg-white py-1 px-2"
              href="/about"
              role="link"
            >
              About & Contact
            </a>
          </p>
        </div>
      </footer>
      <div
        class="fixed bottom-4 right-4 z-20"
      />
    </main>
  </div>
</DocumentFragment>
`;
